opent2t.p.hub/com.wink.hub/js/thingTranslator.js

```diff
@@ -108,6 +110,42 @@ class Translator {
         // Make the async request
         return this._makeRequest(requestPath, 'PUT', putPayloadString);
     }
+
+    /**
+     * Refreshes the OAuth token for the hub by sending a refresh POST to the wink provider
+     */
+    refreshAuthToken(authInfo)
+    {
+        console.log("In refreshAuthToken for WINK hub..");
```

Comment:
I can remove it - we are inconsistent though -- onboarding for eg we do have a similar log statement. do we have a story yet for replacing console.logs with 'wrapperLibraryForTelemetry.log' ?